sql/api/pom.xml

```diff
@@ -76,6 +76,20 @@
             <groupId>org.apache.arrow</groupId>
             <artifactId>arrow-memory-netty</artifactId>
         </dependency>
+
+        <dependency>
+            <groupId>org.apache.spark</groupId>
+            <artifactId>spark-core_${scala.binary.version}</artifactId>
```

Comment:
Please, remove the dependency. The `api` is used by the client (connect), and they shall not depend to `core`.

Reply:
Okay, I'm having a hard time getting it to compile locally when I remove the test scope dependency on core because `SparkFunSuite` depends on `org.apache.spark.internal.config.Tests.IS_TESTING`, but maybe I am doing something wrong locally. I removed it and hopefully it won't have the issues I was having locally.

Reply:
It looks like it worked so it must have been something local for me!